### ISSUE TITLE
Possibly incorrect auto-discovery config format in writeToMqttHA.sh

### ISSUE THREAD

**Issue by jpaulickcz** [belkop-ghb/LYWSD03MMC#1]
Hi, first of all, thanks for this! With your script I got basic functionality out of the sensor like 5 minutes after I unpacked it!

Where I got stuck was the MQTT Autodiscovery within HomeAssistatnt (the  `writeToMqttHA.sh`), which didn't work for me at all (I never used it AD until today, so didn't know where to look). Anyway, I've found the culprit:

The `writeToMqttHA.sh` currently publishes the config with the topic in the following format:

```bash
mosquitto_pub (...) -t homeassistant/TEST_temperature/config
```

But according to [current documentation](https://www.home-assistant.io/integrations/mqtt/#mqtt-discovery), the discovery topic needs to follow this format 
 `<discovery_prefix>/<component>/<object_id>/config`.
 
 Basically there needs to be a component specified (in this case, it's a `sensor`) between the discovery prefix (`homeassistant`) and the object ID. Therefore the working version is:

```bash
mosquitto_pub (...) -t homeassistant/sensor/TEST_temperature/config
```

Hope this helps someone.
